File: src/main/java/no/nav/boot/conditionals/OnProdCondition.java
package no.nav.boot.conditionals;

import org.springframework.core.type.AnnotatedTypeMetadata;

public class OnProdCondition extends OnClusterCondition {

    @Override
    protected Cluster[] clusters(AnnotatedTypeMetadata md) {
        return Cluster.prodClusters();
    }
}

File: src/main/kotlin/no/nav/boot/conditionals/OnProdCondition.kt
package no.nav.boot.conditionals

import no.nav.boot.conditionals.Cluster.Companion.prodClusters
import org.springframework.core.type.AnnotatedTypeMetadata

class OnProdCondition : OnClusterCondition() {
    override fun clusters(md: AnnotatedTypeMetadata) = prodClusters()
}